### DISCUSSION TITLE
Ella Kaye - Welcome to my Quarto website!

### DISCUSSION THREAD

**Discussion by giscus** [EllaKaye/ellakaye-quarto#4] [General]
# Ella Kaye - Welcome to my Quarto website!

Resources I found inspiring and useful when building my new site, plus notes on my customisations

https://ellakaye.co.uk/posts/2022-12-11_welcome-quarto/

**Comment by reddylee** [comment]
Cool!

**Reply by EllaKaye** [reply]
Thank you!

**Comment by reddylee** [comment]
Can we add a comment function which does not need github account?

**Reply by EllaKaye** [reply]
I haven't tried it myself, but I think this is possible with [hypothes.is](https://web.hypothes.is). See <https://quarto.org/docs/output-formats/html-basics.html#commenting> for details.
